README.md

```diff
@@ -304,8 +304,33 @@ By default, requests are retried once and warnings are logged in case of hitting
 };
 ```
 
+To opt-out of this feature:
+
+```js
+new Octokit({ throttle: { enabled: false } })
+```
+
 Throttling in a cluster is supported using a Redis backend. See [`@octokit/plugin-throttling` Clustering](https://github.com/octokit/plugin-throttling.js/#clustering)
 
+</td>
+  </tr>
+   <tr>
+      <th>
+        <code>retry</code>
+      </th>
+      <td>
+        <code>Object</code>
+      </td>
+      <td>
+
+Failed requests are retried by default.
```

Comment:
I'd reference the plugin with its more exhausting documentation for the `retry` parameter

```suggestion
`Octokit` implements request retries using [`@octokit/plugin-retry`](https://github.com/octokit/plugin-retry.js/#readme)
```